src/qt/clientmodel.cpp

```diff
@@ -88,6 +89,16 @@ QDateTime ClientModel::getLastBlockDate() const
     return QDateTime::fromTime_t(Params().GenesisBlock().GetBlockTime()); // Genesis block's time of current network
 }
 
+size_t ClientModel::getMempoolSize() const
```

Comment:
Thanks! Right, .. the map's size() itself is `size_t`, but the CTxMempool uses `long size()` in its interface.
Fixed.